kit/inputs/src/UIMaterialTextView/useClearButton.tsx

```diff
@@ -22,9 +25,11 @@ export function useClearButton(
                     onPress={clear}
```

Comment:
@SAnatoliiS btw, have you found out why this button wasn't pressing on Android device?

Reply:
Yes, I fixed it.